app/1.0/docs/devguide/data-system.md

```diff
@@ -539,7 +539,7 @@ The contents of the event vary depending on the change.
 
 Native elements like `<input>` don't provide the change notification events that Polymer uses for
 upward data flow. To support two-way data binding of native input elements, Polymer lets you
-associate a _custom change notification event_ with a data binding. For example, when binding to a
```

Comment:
Again, we're essentially introducing new terminology here, and the italics are appropriate.